src/components/Article.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ props }) => {
+  const { title, text, date } = props;
+
+  return (
+    <>
+      <h1>
+        {title}
+      </h1>
+      <span>
+        {date}
+      </span>
+      <p>
+        {text}
+      </p>
```

Comment:
Since only one line you could use like:
```suggestion
      <h1>{title}</h1>
      <span>{date}</span>
      <p>{text}</p>
```

Reply:
Done!

I did my initial layout that way as I thought it would look better if it were in the same layout as pure HTML would be.

I've update my code though with your suggestion.